File: docs/components_page/components/nav/pill.R
library(dashBootstrapComponents)

nav <- dbcNav(
  list(
    dbcNavItem(dbcNavLink("Active", active = TRUE, href = "#")),
    dbcNavItem(dbcNavLink("A link", href = "#")),
    dbcNavItem(dbcNavLink("Another link", href = "#")),
    dbcNavItem(dbcNavLink("Disabled", disabled = TRUE, href = "#"))
  ),
  pills = TRUE
)

File: docs/components_page/components/nav/pill.jl
using DashBootstrapComponents

nav = dbc_nav(
    [
        dbc_navitem(dbc_navlink("Active", active = true, href = "#")),
        dbc_navitem(dbc_navlink("A link", href = "#")),
        dbc_navitem(dbc_navlink("Another link", href = "#")),
        dbc_navitem(dbc_navlink("Disabled", disabled = true, href = "#")),
    ],
    pills = true,
);
